src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -62,12 +62,35 @@ internal sealed override bool RequiresCompletion
 
         internal sealed override bool HasComplete(CompletionPart part)
         {
-            return state.HasComplete(part);
+            return _state.HasComplete(part);
         }
 
         internal override void ForceComplete(SourceLocation locationOpt, CancellationToken cancellationToken)
         {
-            state.DefaultForceComplete(this);
```

Comment:
>state.DefaultForceComplete(this); [](start = 12, length = 33)

It looks like there is another consumer of this helper

Reply:
Good catch and it looks like that has the same issue. Let me get antother PR out.